cpp/src/parquet/properties.h

```diff
@@ -145,6 +145,15 @@ static constexpr Encoding::type DEFAULT_ENCODING = Encoding::UNKNOWN;
 static const char DEFAULT_CREATED_BY[] = CREATED_BY_VERSION;
 static constexpr Compression::type DEFAULT_COMPRESSION_TYPE = Compression::UNCOMPRESSED;
 static constexpr bool DEFAULT_IS_PAGE_INDEX_ENABLED = false;
+static constexpr int32_t DEFAULT_BLOOM_FILTER_NDV = 1024 * 1024;
+static constexpr double DEFAULT_BLOOM_FILTER_FPP = 0.05;
+
+struct PARQUET_EXPORT BloomFilterOptions {
```

Comment:
I think current impl is more clear, since user should always understand what he/she is doing when using bf, rather than enable it with a default value

Reply:
IMO, it is fine to provide a default FPP but users are rarely expected to use the default NDV.